db/migrate/20250519201017_add_price_group_to_estimates.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class AddPriceGroupToEstimates < ActiveRecord::Migration[7.0]
+  class Estimate < ApplicationRecord
+    belongs_to :price_group
+    belongs_to :facility
+  end
+
+  def change
+    add_reference :estimates, :price_group
+
+    # Populate the Price group for each existing Estimate.
+    # Just using the first price group for the facility, as we don't have any estimates in prod yet, so we don't need to worry about the price group.
+    Estimate.find_each do |estimate|
+      estimate.update(price_group: estimate.facility.price_groups.first)
+    end
+  end
```

Comment:
Maybe we can do it in an up/down block and removing the reference in the down block